Sources/UserModule/Adapters/PushToken+Adapters.swift

```diff
@@ -0,0 +1,57 @@
+import FeatherComponent
+import FeatherModuleKit
+import FeatherPush
+import FeatherValidation
+import NanoID
+import UserModuleDatabaseKit
+import UserModuleKit
+
+extension [User.PushToken.Model] {
+
+    public func toPushRecipient() -> [Recipient] {
+        var array: [Recipient] = []
+        for item in self {
+            array.append(
+                .init(
+                    token: item.token,
+                    platform: Platform.custom(item.platform)
+                )
+            )
+        }
+        return array
+    }
+
+}
```

Comment:
I would change this for an init function implementation to make sync with the other adapter implementations, like this:
```suggestion
extension Recipient {
   init(model: User.PushToken.Model) {
    self.init(model... etc.)
   }
}

extension [Recipient] {
    init(models: [Model]) {
        self = models.map { Recipient(model: $0) }
    }
}
```
of course after this change you need to change the lines in PushController.swift from 'recipients = tokenList.toPushRecipient()' to 'recipients = .init(models: tokenList)'